registration-processor/pre-processor/registration-processor-packet-classifier-stage/src/main/java/io/mosip/registration/processor/stages/packetclassifier/tagging/impl/IDObjectDataAvailabilityTagGenerator.java

```diff
@@ -141,6 +141,8 @@ private String getValueBasedOnType(String fieldName, FieldDTO fieldDTO) throws J
         }
         String type = fieldDTO.getType();
         String value = fieldDTO.getValue();
+        
+        String tagLanguage = classifierUtility.getTagLanguageBasedOnFieldDTO(value);
```

Comment:
If the value is a numerical field, how will this method handle ?

Reply:
moved this method call inside simple type switch case...